src/main/java/TaskManager.java

```diff
@@ -0,0 +1,239 @@
+import java.io.IOException;
+import java.util.ArrayList;
+
+public class TaskManager {
+
+    //These indexes are the start index of the string which has
+    //the unnecessary part removed
+    private static final int TODO_START_INDEX = 4;
+    private static final int DEADLINE_START_INDEX = 8;
+    private static final int EVENT_START_INDEX = 5;
+    private static final int DELETE_NUMBER_INDEX = 6;
+    private static final int DONE_NUMBER_INDEX = 4;
+    private static final int TASK_DESCRIPTION_INDEX = 0;
+    private static final int TASK_DATE_INDEX = 1;
+
+    private static final ArrayList<Task> tasks = new ArrayList<>();
+
+    //only for adding tasks when program is already runnning
+    //not for adding the tasks when loaded into the syste
+    public void addTaskPlusException(TaskEnum taskType, String userInput) {
+        try {
+            String userInputWithoutTaskCommand = removeTaskCommand(taskType, userInput);
+            addTask(taskType, userInputWithoutTaskCommand, false);
+
+        } catch (BlankDescriptionException e) {
+            //TODO differentiate tasks
+            Duke.printlnTab("☹ OOPS!!! The description of a task cannot be empty.");
+            Duke.printDivider();
+        }
+    }
+
+    //command keyword removed eg. "todo clean room"  -> "clean room"
+    private String removeTaskCommand(TaskEnum taskType, String userInput) throws BlankDescriptionException {
+        String strippedUserInput = "";
+
+        switch (taskType) {
+        case TODO:
+            strippedUserInput = userInput.substring(TODO_START_INDEX).strip(); // remove "todo" from userInput
+            break;
+        case DEADLINE:
+            strippedUserInput = userInput.substring(DEADLINE_START_INDEX).strip(); // remove "deadline" from userInput
+            break;
+        case EVENT:
+            strippedUserInput = userInput.substring(EVENT_START_INDEX).strip(); // remove event
+            break;
+        }
+
+        if (strippedUserInput.isEmpty()) {
+            throw new BlankDescriptionException();
+        }
+        return strippedUserInput;
+    }
+
+    private void addTask(TaskEnum taskType, String userInputWithoutTaskCommand, boolean isDone) {
+
```

Comment:
You have a blank line here right after the method name, but in other methods there is no blank line after the method name. Keep the line breaks consistent.